bitnami/prestashop/8/debian-11/docker-compose.yml

```diff
@@ -1,5 +1,6 @@
 version: '2'
 services:
+  # New change
```

Comment:
Please, remove the trigger in docker-compose.yml and I'll happily accept the PR. Your changes LGTM 🚀